### ISSUE TITLE
Move gemfile code to main block

### ISSUE THREAD

**Issue by fulldecent** [rubygems/rubygems.org#3323]
<img width="1567" alt="Screenshot 2023-01-05 at 13 28 51" src="https://user-images.githubusercontent.com/382183/210854048-9c586b6e-7528-4bc9-a736-cff5612fbaa4.png">

Currently, truncation on the right side bar requires you to copy/paste code sight-unseen.

Moving into into the main content area will allow more room to see the full text.

(Issue moved from https://github.com/rubygems/rubygems.org/issues/3318#issuecomment-1367865608)

**Comment by simi** [comment]
Thanks for opening this issue. Do you have plan to open PR with given change?

It seems the key would be to move following block

https://github.com/rubygems/rubygems.org/blob/273acd5421dd266ee37d9159958e2bbf84d28b2c/app/views/rubygems/_aside.html.erb#L22-L39

into 

https://github.com/rubygems/rubygems.org/blob/273acd5421dd266ee37d9159958e2bbf84d28b2c/app/views/rubygems/show.html.erb.

@jenshenny any other ideas?

**Comment by rubyFeedback** [comment]
I guess the question is somewhat related to how to structure the UI of the homepage of gems. On my widescreen monitor everything is quite condensed in the center; perhaps only 33% or so is occupied with content specific to the gem at hand, so 67% or so is kind of empty. This may be useful on mobile devices but on my desktop system it is a bit of a waste of space.

I would perhaps make the display a bit larger (say, from 33% occupied, to 40% or something like that, so a fairly small change still; that way the center becomes a bit wider and can accomodate more content), and then, in regards to the suggestion, perhaps consider moving "Gemfile" and "Install" onto the middle-right side, that is, put it next to "Runtime dependencies" or something like that. I have not thought through all implications of this, UI-wise, though, and I have no real preference either way. To me, functionality is more important at this point in time.

**Comment by jenshenny** [comment]
I'm open to moving the block to the main section and increasing the display in general as @rubyFeedback suggests. 

Some other ideas I can think of is making the fields scrollable so the user can view the full text, or getting rid of the text preview and have `Add to Gemfile` and `Gem install` headers with the button to copy (isn't as user friendly). I don't have any strong opinions for which idea gets executed.

**Comment by simi** [comment]
I agree on both points (utilizing more than 940px on the main content and moving the gem blocks). Let's focus in this cose on moving the gem block. I did quick test with following result.

![image](https://user-images.githubusercontent.com/193936/212169304-6cc96253-ccad-4f3d-9a49-92454489970d.png)

I would like to also check if we can recommend using `bundle add` already. I don't remember when that was introduced.

See attached diff. @fulldecent would you like to contribute this? Feel free to use my changes as a base (you'll need also to change locales since some relative I18n keys have changed).

```diff
diff --git a/app/views/rubygems/_aside.html.erb b/app/views/rubygems/_aside.html.erb
index 6d3fb4d8..7f56c004 100644
--- a/app/views/rubygems/_aside.html.erb
+++ b/app/views/rubygems/_aside.html.erb
@@ -19,25 +19,6 @@
     </h2>
   </div>
 
-  <% if @latest_version.indexed %>
-    <h2 class="gem__ruby-version__heading t-list__heading">
-      <%= t '.bundler_header' %>:
-      <div class="gem__code-wrap">
-        <input type="text" class="gem__code" id="gemfile_text" value="<%= @latest_version.to_bundler %>" readonly="readonly">
-        <span class="gem__code__icon" id="js-gem__code--gemfile" data-clipboard-target="#gemfile_text">=</span>
-        <span class="gem__code__tooltip--copy"><%= t('.copy_to_clipboard') %></span>
-        <span class="gem__code__tooltip--copied"><%= t('.copied') %></span>
-      </div>
-    </h2>
-    <h2 class="gem__ruby-version__heading t-list__heading">
-      <%= t '.install' %>:
-      <div class="gem__code-wrap">
-        <input type="text" class="gem__code" id="install_text" value="<%= @latest_version.to_install %>" readonly="readonly">
-        <span class="gem__code__icon" id="js-gem__code--install" data-clipboard-target="#install_text">=</span>
-      </div>
-    </h2>
-  <% end %>
-
   <h2 class="gem__ruby-version__heading t-list__heading">
     <%= pluralized_licenses_header @latest_version %>:
     <span class="gem__ruby-version">
diff --git a/app/views/rubygems/show.html.erb b/app/views/rubygems/show.html.erb
index 044fdd3f..07f037bd 100644
--- a/app/views/rubygems/show.html.erb
+++ b/app/views/rubygems/show.html.erb
@@ -18,6 +18,23 @@
         <div id="markup" class="gem__desc">
           <%= simple_markup(@latest_version.info) %>
         </div>
+
+        <h2 class="gem__ruby-version__heading t-list__heading">
+          <%= t '.bundler_header' %>:
+          <div class="gem__code-wrap">
+            <input type="text" class="gem__code" id="gemfile_text" value="<%= @latest_version.to_bundler %>" readonly="readonly">
+            <span class="gem__code__icon" id="js-gem__code--gemfile" data-clipboard-target="#gemfile_text">=</span>
+            <span class="gem__code__tooltip--copy"><%= t('.copy_to_clipboard') %></span>
+            <span class="gem__code__tooltip--copied"><%= t('.copied') %></span>
+          </div>
+        </h2>
+        <h2 class="gem__ruby-version__heading t-list__heading">
+          <%= t '.install' %>:
+          <div class="gem__code-wrap">
+            <input type="text" class="gem__code" id="install_text" value="<%= @latest_version.to_install %>" readonly="readonly">
+            <span class="gem__code__icon" id="js-gem__code--install" data-clipboard-target="#install_text">=</span>
+          </div>
+        </h2>
       </div>
     <% else %>
       <div class="t-body">

```

**Comment by hsbt** [comment]
I'm 👍 to this change.

**Comment by simi** [comment]
@fulldecent @rubyFeedback feel free to review https://github.com/rubygems/rubygems.org/pull/3492 :pray:

**Comment by simi** [comment]
This was fixed by #3492 and already deployed. Thanks to all involved!
